src/Item/Item.php

```diff
@@ -10,8 +10,8 @@ public function __construct(array $data)
     {
         $this->data = $data;
 
-        $this->data['id'] = $this->data['@id'];
-        if (preg_match('/(\d+)$/', $this->data['@id'], $matches)) {
+        $this->data['id'] = $this->data['entityId'];
+        if (preg_match('/(\d+)$/', $this->data['entityId'], $matches)) {
```

Comment:
Consider adding a comment on what's going on here, e.g.

```suggestion
        // If the ID ends with an integer, use the integer as the ID.
        if (preg_match('/(\d+)$/', $this->data['entityId'], $matches)) {
```

Reply:
Added